File: src/test/kotlin/com/leetcode/P3137Test.kt
package com.leetcode

import org.hamcrest.MatcherAssert.assertThat
import org.hamcrest.Matchers.`is`
import org.junit.jupiter.api.Test

class P3137Test {
    val sut = P3137()

    @Test
    fun `example 1`() {
        val operations = sut.minimumOperationsToMakeKPeriodic("leetcodeleet", 4)
        assertThat(operations, `is`(1))
    }

    @Test
    fun `example 2`() {
        val operations = sut.minimumOperationsToMakeKPeriodic("leetcoleet", 2)
        assertThat(operations, `is`(3))
    }

    @Test
    fun `example 3`() {
        val operations = sut.minimumOperationsToMakeKPeriodic("a", 1)
        assertThat(operations, `is`(0))
    }
}

File: src/python3/com/leetcode/p3137_test.py
import unittest

import p3137


class Test(unittest.TestCase):
    sut = p3137.Solution()

    def test_1(self):
        operations = self.sut.minimumOperationsToMakeKPeriodic("leetcodeleet", 4)
        self.assertEqual(operations, 1)

    def test_2(self):
        operations = self.sut.minimumOperationsToMakeKPeriodic("leetcoleet", 2)
        self.assertEqual(operations, 3)
